### ISSUE TITLE
Title and role attributes are missing for functional control on Editor screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#798]
**Describe the bug**
Title and role attributes are missing for functional control on Editor screen. When we edit a page for a site, then there is no label/title displayed when we hover over "Close" button. Also, when we hover over different tabs such as "Content", " Layout" or "Style" then also no label is displayed. 

**To Reproduce**
1. Go to Editor screen (http://localhost:9992/cm/app/?view=editor&pathType=unknown)
2. Then select any site and go to one of its pages and edit that.
3. There is no label/title displayed when we hover over "Close" button
4. When we hover over different tabs such as "Content", " Layout" or "Style" then also no label is displayed.
